docTemplates/README.md

```diff
@@ -0,0 +1,110 @@
+<!--
```

Comment:
This file is generated upon request. Should not be committed to version control. Please update the .gitignore https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.gitignore.temp